packages/protocol-kit/src/utils/transactions/gas.ts

```diff
@@ -111,8 +110,8 @@ export async function estimateTxGas(
   data: string,
   operation: OperationType
 ): Promise<string> {
-  let txGasEstimation = BigNumber.from(0)
-  const safeAddress = safeContract.getAddress()
+  let txGasEstimation = 0
```

Comment:
Number migration was not achieved correctly. Ethers now don't need the BigNumber lib, because there is native BigInt in Javascript, but should be used accordingly

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt